spec/models/quarter_spec.rb

```diff
@@ -175,4 +260,32 @@
       expect(quarter.quarter_rank).to eq 4
     end
   end
+  describe "#year" do
+    context "with a code of 201415" do
+      let(:quarter) { described_class.new(201415) }
+      it "returns 2014" do
+        expect(quarter.year).to eq 2014
+      end
+    end
+    context "with a code of 201425" do
+      let(:quarter) { described_class.new(201425) }
+      it "returns 2015" do
+        expect(quarter.year).to eq 2015
+      end
+    end
+  end
+  describe '#year_code' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.